packages/app/src/settings/services/settings.service.ts

```diff
@@ -32,7 +32,11 @@ export class SettingsService {
   }
 
   updateSettings({ key, value }: SettingsUpdateOption): SettingsValue {
-    this.store.set(key, value)
+    if (value !== undefined) {
+      this.store.set(key, value)
+    } else {
+      this.store.delete(key)
+    }
```

Comment:
Maybe update should only update and another function/method should be delivered to do deleting?